adafruit_ble/services/standard/standard.py

```diff
@@ -29,10 +29,10 @@
 
 import time
 
-from ..core import Service
-from ...core.uuid import StandardUUID
+from .. import Service
+from ...uuid import StandardUUID
 from ..characteristics.string import StringCharacteristic
-from ..characteristics.core import StructCharacteristic
+from ..characteristics import StructCharacteristic
```

Comment:
FYI -- each of these needs one more "."   ```...characteristics```

Reply:
Still returns all 0's ,but with that change, it does not throw any errors ;-)

Reply:
Oops, I will push another change, but what returns all 0's?

Reply:
I've been running this -- modified from your old version  
```
"""CTS demo
"""

import board
import displayio
import terminalio
import time

import adafruit_displayio_ssd1306
from adafruit_display_text import label
from adafruit_ble import BLERadio
from adafruit_ble.advertising.standard import ProvideServicesAdvertisement
from adafruit_ble.services.standard.standard import CurrentTimeService

ble = BLERadio()
cts = CurrentTimeService()
advertisement = ProvideServicesAdvertisement(cts)



displayio.release_displays()

i2c = board.I2C()
display_bus = displayio.I2CDisplay(i2c, device_address=0x3c)
display = adafruit_displayio_ssd1306.SSD1306(display_bus, width=128, height=32)

FONT = terminalio.FONT
WHITE = 0xffffff
CONNECTING = "Connecting..."
text = CONNECTING

text_area = label.Label(FONT, max_glyphs=50, text=text, color=WHITE)

text_area.x = 0
text_area.y = 10


display.show(text_area)

while True:
    text_area.text = CONNECTING
    ble.start_advertising(advertisement)
    while not ble.connected:
        pass

    while ble.connected:
        print(cts.struct_time)
        year, month, day, hour, minute, second, _, _, _ = cts.current_time
        text_area.text = '{}-{}-{} {:02}:{:02}:{:02}'.format(year, month, day, hour, minute, second)
        time.sleep(5)

```

I don't get the pairing option on Settings on my iPhone, but if I connect via the BLE Conect App it connects and but the time struct is all 0's 
```
Press any key to enter the REPL. Use CTRL-D to reload.
Adafruit CircuitPython 5.0.0-alpha.5-84-g4c55431bb on 2019-11-14; Adafruit Feather nRF52840 Express with nRF52840
>>> import cts
struct_time(tm_year=0, tm_mon=0, tm_mday=0, tm_hour=0, tm_min=0, tm_sec=0, tm_wday=-1, tm_yday=-1, tm_isdst=-1)
```

Reply:
This does not have to be resolved for this PR -- no need to hold things up.